packages/components/src/components/search/search.tsx

```diff
@@ -119,6 +125,16 @@ export class Search implements ComponentInterface {
      */
     @Prop() disabled: boolean = false;
 
+    /**
+     * The Mapbox Session Token used for getting Mapbox autocomplete suggestions.
+     */
+    @Prop() sessionToken: string;
```

Comment:
Clarification: Is this used outside this component? Can we generate the session token inside this component?

Reply:
Yes, this is used outside of the component as well. The reason for that is that we need to use the same session token in the search component `search.tsx` and also in the `MapboxPlacesHandler.js` file, when we retrieve the geometry. These two have to be the same, therefore we created an atom which stores the value, and we pass it down to the component.

Reply:
That's true. I was just hoping we could avoid the prop 😅